src/Mesh/Interpolation.jl

```diff
@@ -51,34 +42,51 @@ struct InterpolationBrick{
 
     xbnd::FTA2     # domain bounds, [2(min/max),ndim]
 
-    x1g::FTV # 1D interpolation grid in x1 direction
+    x1g::FTV       # 1D interpolation grid in x1 direction
     x2g::FTV
     x3g::FTV
 
-    ξ1::FTVD # unique ξ1 coordinates of interpolation points within each element
-    ξ2::FTVD # unique ξ2 coordinates of interpolation points within each element
-    ξ3::FTVD # unique ξ3 coordinates of interpolation points within each element
+    ξ1::FTVD       # unique ξ1 coordinates of interpolation points within each element
+    ξ2::FTVD       # unique ξ2 coordinates of interpolation points within each element
+    ξ3::FTVD       # unique ξ3 coordinates of interpolation points within each element
 
-    flg::UI8AD # flags when ξ1/ξ2/ξ3 interpolation point matches with a GLL point
+    flg::UI8AD     # flags when ξ1/ξ2/ξ3 interpolation point matches with a GLL point
 
-    fac::FTVD # normalization factor
+    fac::FTVD      # normalization factor
 
-    x1i::UI16VD # interpolation grid indices of interpolation points within each element
+    x1i::UI16VD    # interpolation grid indices of interpolation points within each element
     x2i::UI16VD
     x3i::UI16VD
 
-    offset::TVD  # offsets for each element for v
+    offset::TVD    # offsets for each element for v
 
-    m1_r::FTVD  # GLL points
-    m1_w::FTVD  # GLL weights
-    wb::FTVD    # Barycentric weights
+    m1_r::FTVD     # GLL points
+    m1_w::FTVD     # GLL weights
+    wb::FTVD       # Barycentric weights
     # MPI setup for gathering interpolated variable on proc # 0
     Nel_all::I32V
 
-    x1i_all::UI16V
-    x2i_all::UI16V
-    x3i_all::UI16V
+    x1i_all::UI16VD
+    x2i_all::UI16VD
+    x3i_all::UI16VD
     #--------------------------------------------------------
+    """
+        InterpolationBrick(grid::DiscontinuousSpectralElementGrid{FT},
```

Comment:
It doesn't seem like docstrings work correctly on inner constructors (see https://github.com/JuliaLang/julia/issues/16730): either make it an outer constructor, or move it to the `struct` itself.

(same for `InterpolationCubedSphere`)